### DISCUSSION TITLE
Limit Project member to become Project Owner & from creating new projects.

### DISCUSSION THREAD

**Discussion by punitdiwan** [appwrite/appwrite#3261] [Ideas]
**Problem**
        When new user gets added to the project from the Project member tab, after email verification, they become project owners and enjoys same privileges as the first Project Member.  Moreover, they can create new projects. In fact they become platform owns.

**Solution**
	 There are various possibilities which can be managed in the this platform. We can have two types of Users who can log in into the console. One is Platform Owner and other is project Owner. Platform Owner can create multiple projects while Project Owners can only manage the particular project of which he is member off.  Similarly in case of Leaving a project, if he is not platform owner and 
has only one project of which his membership has been revoked, the user itself should get deleted.

Lot of thing can be done in the area of authorization. To keep things concise at this moment, we can limit the user to that project. What I mean we can restrict the new member from creating new projects. We can add a checkbox, that allows to create further projects. If radio button is unchecked, the won’t be able to create the project. He will be limited to the assigned projects.  Later we can further limit his roles.

Please add something If I have missed.

Regards
Pnit

**Comment by eldadfux** [comment]
Managing console roles is definitely something that should be on our roadmap. Would love to get more community feedback on this feature before we start to work on design and implementation.

**Reply by stnguyen90** [reply]
I think it would make sense to have invited members be admins on the project, but not allow them to create new projects or remove the original owner.

**Comment by gewenyu99** [comment]
👋 Hi there @punitdiwan 

The current behavior of Appwrite console is like this:
- When members are invited, they can only see the projects they're invited to.
- When members are invited, they can create new projects, that they own and others cannot see.

I think more granular control is definitely something we'll need down the road. I am going to turn this into a discussion, so more people can see it and share their thoughts.

Cheers~
